tests/test_fit_result.py

```diff
@@ -68,27 +79,88 @@ def test_no_constraint(minimizer, fit_kwargs, expected_par, expected_std, expect
          {'objective_value': -float('inf'), 'log_likelihood': float('inf'), 'likelihood': float('inf')})
     ])
 def test_constraints(minimizer, fit_kwargs, expected_par, expected_std, expected_obj, expected_got):
+    """
+    Tests the FitResults from fitting a simple model with Le as constraint
+    using several different minimizers and objectives.
+    """
+
+    model=Model({y: a * x ** b})
+    constraints = [
+        Le(a, b)
+    ]
+    fit = Fit(**fit_kwargs, minimizer=minimizer, model=model, constraints=constraints)
+    fit_result = fit.execute()
+    _run_tests(fit_result, expected_par, expected_std, expected_obj, expected_got)
+
+
+@pytest.mark.parametrize('minimizer',
+    # Added None to check the defaul objective
+    # Removed TrustConstr, because it cannot handle CNM
+    subclasses(ConstrainedMinimizer) - {TrustConstr} | {None}
+)
+@pytest.mark.parametrize('fit_kwargs, expected_par, expected_std, expected_obj, expected_got',
+    [
+        # No special objective
+        (dict(x=np.linspace(1, 10, 10), y=3 * np.linspace(1, 10, 10) ** 2), {a: 2.152889, b: 2.152889}, 
+         {a: 0.23715, b: 0.05076}, LeastSquares, {'r_squared': 0.99791, 'objective_value': 98.83587, 'chi_squared': 197.671746}),
```

Comment:
For this one and others you can consider taking some more vertical space, and spread each case out over 5 lines.